src/assets/styles/10-settings/_typography.scss

```diff
@@ -21,12 +21,15 @@
   --text-base-size-px: 16;
 
   // font size - non-responsive body text
-  --body-xl: calc(20/16 * var(--text-base-size));     // Large
-  --body-lg: var(--text-base-size);                   // Regular
-  --body-md: calc(14/16 * var(--text-base-size));     // Small
-  --body-sm: calc(13/16 * var(--text-base-size));     // Smaller
-  --body-xs: calc(11/16 * var(--text-base-size));     // Smallest
-  --body-xxs: calc(10/16 * var(--text-base-size));    // 
+  --body-xl: calc(24/16 * var(--text-base-size));     // Large
+  --body-lg: calc(20/16 * var(--text-base-size));     // Regular
+  --body-md: var(--text-base-size);                   // Small
+  --body-sm: calc(14/16 * var(--text-base-size));     // Smaller
+  --body-xs: calc(12/16 * var(--text-base-size));     // Smallest
+  /**
+   * We don't have this size in design system
+   */
```

Comment:
As this comment is on a single line, I'd suggest using a single line comment format (`// we don't...`) rather than a block comment, just for tidiness 🙂